### DISCUSSION TITLE
Automatically booting to WSL?

### DISCUSSION THREAD

**Discussion by leo-petrucci** [vercel/hyper#6000] [Q&A] [answered]
Any way to get Hyper to automatically start WSL when opened?

As of now I manually need to type `wsl` every time I open a new tab/window.

**Comment by LabhanshAgrawal** [comment, marked as answer]
you can configure the `shell` in the config file `ctrl+,`

**Reply by LabhanshAgrawal** [reply]
e.g. `C:\\Windows\\System32\\wsl.exe`

**Reply by leo-petrucci** [reply]
Also had to change `shellArgs`:
```
    shellArgs: [],
```

Thank you!
